pkg/platformconfig/types.go

```diff
@@ -181,3 +181,9 @@ const (
 
 	DefaultServiceType = corev1.ServiceTypeClusterIP
 )
+
+const DefaultStreamMonitoringWebapiURL = "http://v3io-webapi:8081"
+
+type StreamMonitoringConfig struct {
+	WebapiURL string `json:"webapiURL,omitempty"`
```

Comment:
add concurrency value (default 10) for how many get shard lags requests can be fired at once during get stream shards